pkg/config/config.go

```diff
@@ -610,6 +613,14 @@ func (c *APIConfig) Validate(onExecution bool) error {
 	if c.GRPCMaxRecvMsgSize <= 0 {
 		c.GRPCMaxRecvMsgSize = defaultGRPCMaxMsgSize
 	}
+	// merge deprecated option HostIP with new option HostIPs
+	// set HostIP to empty to have it no longer printed in configs
+	if c.HostIP != "" {
+		if len(c.HostIPs) < 2 {
```

Comment:
What should we do when `len(HostIPs) >= 2` and `HostIP` is set?

Reply:
fail, because we unconditionally append HostIP to HostIPs.